platform-includes/getting-started-sourcemaps/javascript.react-router.mdx

```diff
@@ -1,9 +1,43 @@
-## Add Readable Stack Traces to Errors
+## Source Maps Upload
 
-By default, React Router will minify your JavaScript and CSS files in production. This makes it difficult to debug errors. To make debugging easier, you can generate source maps and upload them to Sentry.
+Update `vite.config.ts` to include the `sentryReactRouter` plugin and also add your `SentryReactRouterBuildOptions` config options to the vite config (this is required for uploading source maps at the end of the build):
```

Comment:
This sentence is quite long, I would break it into two.
```suggestion
Update `vite.config.ts` to include the `sentryReactRouter` plugin. Also add your `SentryReactRouterBuildOptions` config options to the Vite config (this is required for uploading source maps at the end of the build):
```